.github/workflows/run-tests.yml

```
@@ -22,21 +21,27 @@ jobs:
         include:
           - perl: "5.22.4"
             irods: "4.2.7"
+            client_image: "ghcr.io/wtsi-npg/ub-18.04-irods-clients-4.2.11:latest"
```

```suggestion
            client_image: "ghcr.io/wtsi-npg/ub-18.04-irods-clients-4.2.7:latest"
```
Like the idea of shifting to testing 4.2.11 clients with 4.2.7 server - but is this your intention?

---

Ah, this is where we get baton, and that (suitable version anyway) is not in this old container I think

---

It's intentional to test 4.2.11 clients against the 4.2.7 server because that combination _might_ happen during the server update (but hopefully not)